qupath-core/src/main/java/qupath/lib/images/servers/SlicedImageServer.java

```diff
@@ -0,0 +1,113 @@
+package qupath.lib.images.servers;
+
+import qupath.lib.io.GsonTools;
+import qupath.lib.regions.RegionRequest;
+
+import java.awt.image.BufferedImage;
+import java.io.IOException;
+import java.util.Map;
+
+/**
+ * ImageServer that treats a particular set of z-slices and timepoints of another ImageServer
+ * as a full image.
+ */
+public class SlicedImageServer extends TransformingImageServer<BufferedImage> {
+
+    private final ImageServerMetadata metadata;
```

Comment:
What do you think of including `zStep` and `tStep` values, with defaults of 1?

"No" is a valid answer here, but I wonder if subsampling may sometimes be desirable. *However* it may not be and you'd need to update the pixel calibration information if this is supported... so maybe best skip it for now.

Reply:
Yes. I can include that in a future PR

Reply:
Or instead of the `start`, `step`, and `end`, couldn't we simply give a list of indices to consider?

Reply:
Sorry, I missed this question originally...

The trouble with giving a list of indices:
1. it's harder to write a reusable command that gives a list of indices whenever the number of z-slices can change
2. often, we will have a z-spacing value in µm; if we have a step size, we can scale the z-spacing by the step size to get the z-spacing value to store in the output metadata. But if someone selects slices 1, 2, 4, 10 then the resulting z-stack will have different spacing between slices. This isn't generally expected, and means that a single z-spacing value can no longer be used.